packages/iTwinUI-react/src/core/Table/Table.test.tsx

```diff
@@ -2727,6 +2727,156 @@ it('should render action column with column manager', () => {
   expect(actionColumn[3].textContent).toBe('View');
 });
 
+it('should render dropdown menu with specified class and style in action column with column manager', async () => {
```

Comment:
Have I placed the new tests in a correct place in the file? Is there a better place within the file to place these tests?

Reply:
**[SOLVED BY SELF]** Since now there is just one new test instead of nearly four, I think putting the new test here is alright.